doc/content/sqa/index.md

```diff
@@ -1 +1,6 @@
 !template load file=sqa/app_index.md.template app=TMAP8App category=tmap8
+
+TMAP8 employs a continuous integration strategy using [!ac](CIVET); the testing results for
+this version of the documentation is available at the following links:
+
+!civet mergeresults
```

Comment:
```suggestion
# TMAP8 Software Quality Assurance

!sqa records category=_empty_

## Test Results

TMAP8 employs a continuous integration strategy using [!ac](CIVET); the testing results for
this version of the documentation is available at the following links:

!civet mergeresults

!sqa report category=_empty_
```

If you want to include merge results in the SQA page, you should consider not using the application template, and I've laid out the changes required above. Further you'll need to add a bit to the `doc/config.yml` file at the bottom of the `Extensions` section:

```
    MooseDocs.extensions.civet:
        test_results_cache: '/tmp/civet/jobs'
        branch: main
        remotes:
            TMAP8:
                url: https://civet.inl.gov
                repo: idaholab/TMAP8
```

Reply:
I think I just did what you mentioned here.

Reply:
You still need to accept the results I suggested above. The change to `doc/config.yml` was only one part.

Reply:
Added in 078f0f7.